docs/docs/01-why-react.de-DE.md

```diff
@@ -0,0 +1,28 @@
+---
+id: why-react-de-DE
+title: Warum React?
+permalink: why-react-de-DE.html
+---
+React ist eine JavaScript-Bibliothek von Facebook und Instagram für Benutzeroberflächen. Man kann sich React als das **V** in **[MVC](https://de.wikipedia.org/wiki/Model_View_Controller)** vorstellen.
+
+Wir haben React für folgende Aufgabe entwickelt: **umfangreiche Anwendungen zu bauen, deren Daten sich zeitlich ändern**.
+
+## Einfach
+
+Beschreibe, wie sich Deine App zu jedem gewünschten Zeitpunkt präsentieren soll und React kümmert sich um alle Benutzeroberflächen-Änderungen sobald sich die zugrundliegenden Daten ändern.
```

Comment:
If you already have the branch checked out, you can just make the change and use `git commit --amend` to update the old commit.  (If you don't have it checked out, look at https://help.github.com/articles/checking-out-pull-requests-locally/)

Then you can use `git push -f` to forcefully update the branch.